src/test/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformCopyRatioSegmentationIntegrationTest.java

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.hellbender.tools.exome.segmentation;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.exome.ModeledSegment;
+import org.broadinstitute.hellbender.tools.exome.SegmentUtils;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by davidben on 7/25/16.
+ */
+public class PerformCopyRatioSegmentationIntegrationTest extends CommandLineProgramTest {
+    private static final String TOOLS_TEST_DIRECTORY = publicTestDir + "org/broadinstitute/hellbender/tools/exome/";
+    private static final File LOG2_TN_COVERAGE_FILE = new File(TOOLS_TEST_DIRECTORY, "coverages-for-copy-ratio-modeller.tsv" );
+
+    @Test
+    public void testCommandLine() throws IOException {
```

Comment:
Perhaps some comments here indicating that this only tests that output is created and does not check for correctness of the segmentation?